collector/src/main/java/com/bitmovin/analytics/stateMachines/PlayerState.java

```diff
@@ -75,7 +75,7 @@ void onEnterState(PlayerStateMachine machine) {
                 }
             }
 
-            machine.enableHeartbeat();
+//            machine.enableHeartbeat();
```

Comment:
But that would prevent the HeartBeat from running at all (it's required for when we go into the `playing` state)

Reply:
yes but the playing event does have the same actions
so the heartbeat gets disabled by the PlayExit and enabled by the PlayEntered
But with those line the heartbeat would get enabled right after it got disabled
which makes no sense to me right